### PR TITLE
Feature/lp69 outlined text field mail

### PR DESCRIPTION
**Pull request by AndresIvanMedinaH**
**• Task’s name**
OutlinedTextField - Email

    **• Task’s link to navigate to jira**
https://devaptivist.atlassian.net/browse/LP-69?atlOrigin=eyJpIjoiODQ3MmY0MzczNjJiNDYwMjhjMWEzMWYwOWVhMjY3ZWIiLCJwIjoiaiJ9

    **• General task’s description of what was added, deleted or updated in the code.**
As a developer, I must create a component like the image above showing an OutlinedTextField, that implements a label in the top left corner and must have a label with the text = mail

this field will accept the email of the user and must validate that the content received are only valid fields
this TextField belongs to the screen Login

    **• Attachment file of how looks your code in the app or in the preview if it is necessary**

No text  OutlinedTextFieldMail
OutlinedTextFieldMail

2OutlinedTextFieldMail

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/OutLinedTextFieldMail.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.iteneum.designsystem.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.modifier.modifierLocalConsumer
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/OutLinedTextFieldMail.kt]: we don't use this library we can delete it

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/OutLinedTextFieldMail.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.TextFieldValue
+
+
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/OutLinedTextFieldMail.kt]: delete this space please

**Review by carlosph816 — CHANGES_REQUESTED** [review]

**Review by juanislas77 — APPROVED** [review]
Nice work!

**Review by carlosph816 — APPROVED** [review]

**Review by juanislas77 — APPROVED** [review]
Nice Job

**Pull request merged**